code/game/objects/items/toys.dm

```diff
@@ -500,21 +487,71 @@
 	name = "plushie"
 	desc = "An adorable, soft, and cuddly plushie."
 	icon = 'icons/obj/toy.dmi'
-	var/poof_sound = 'sound/weapons/thudswoosh.ogg'
 	attack_verb = list("poofed", "bopped", "whapped","cuddled","fluffed")
 	resistance_flags = FLAMMABLE
+	var/list/poof_sound = list('sound/weapons/thudswoosh.ogg' = 1)
+	var/stuffed = TRUE //If the plushie has stuffing in it
```

Comment:
This is a little confusing, as it can be read as "is it stuffed with something (grenade)"

Perhaps you could rename the var to `has_stuffing` or something?